### ISSUE TITLE
Missing generated types

### ISSUE THREAD

**Issue by Akkuma** [graphql-nexus/nexus-plugin-prisma#343]
Here is my schema & generated photon files https://gist.github.com/Akkuma/de0936c5421c728b057ac3cad662ff79

The output `index.d.ts` is producing `export declare type MenuUpdateManyMutationInput = {};`

When I look through the file all other UpdateManyMutationInput's are properly declared.

I came across this when I passed the result of makeSchema to `apollo-server-express`

```
Error: Input Object type MutationUpdateManyEmployeeFilter must define one or more fields.

Input Object type MutationDeleteManyEmployeeFilter must define one or more fields.

Input Object type MenuUpdateManyMutationInput must define one or more fields.
```

**Comment by Akkuma** [comment]
@timsuchanek I'm using the following code (minus unrelated code)

```js
import { makeSchema } from '@prisma/nexus';
import Photon from '@generated/photon';
import { nexusPrismaPlugin } from '@generated/nexus-prisma';

import { createSchemaTypes } from './prisma';

const photon = new Photon();

const types = createSchemaTypes(photon.dmmf.schema.outputTypes);

const schema = makeSchema({
  debug: true,
  types: [...types, nexusPrisma/**/],
  outputs: {
    schema: join(__dirname, '/prisma/schema.graphql'),
  },
  typegenAutoConfig: {
    sources: [
      {
        source: '@generated/photon',
        alias: 'photon',
      },
    ],
  },
})

const apollo = new ApolloServer({
  schema,
  context: {
    photon: photon
  }
});
```

`createSchemaTypes` is my own creation to simplify getting everything up and running without the tediousness of handwriting it. The reason I created it is that if you write the graphql schema essentially by hand after having written a prisma schema by hand, it seems to offer minimal value over typegraphql & typeorm which kills two birds with one stone, writing a "schema" for your DB and then added a few more decorators onto it for graphql support simultaneously.
```js
import { objectType } from '@prisma/nexus';

export function exposeFromSrc(src, fields, filter = () => true) {
  fields
    .filter(filter)
    .forEach(({name}) => src[name]());
}

const queryVerbage = ['findOne', 'findMany'];
const mutationVerbage = ['create', 'delete', 'update', 'upsert'];

const modelFilter = (blacklist) => ({name}) => !blacklist.includes(name);
const queryFilter = function (blacklist, verbage) {
  const caseInsensitiveBlacklist = blacklist.map(_ => _.toLowerCase());

  return function ({name}) {
    const isTypeBlacklisted = verbage.some(verbage => {
      const type = name.replace(verbage, '');
      return caseInsensitiveBlacklist.includes(type);
    });

    return !isTypeBlacklisted && !blacklist.includes(name);
  }
}
const mutationFilter = queryFilter;


export const PrismaType = {
  Model: Symbol('model'),
  Query: Symbol('query'),
  Mutation: Symbol('mutation'),
}

function match(t, funcs) {
  return funcs[t]();
}

export function expose({ src, fields, type, blacklist = [] }) {
  match(type, {
    [PrismaType.Model]: _ => exposeFromSrc(src, fields, modelFilter(blacklist)),
    [PrismaType.Query]: _ => exposeFromSrc(src, fields, queryFilter(blacklist, queryVerbage)),
    [PrismaType.Mutation]: _ => exposeFromSrc(src, fields, mutationFilter(blacklist, mutationVerbage)),
  });
}

export function objType({ name, type, fields, blacklist = [] }) {
  return objectType({
    name: name,
    definition(t) {
      if (name === 'BatchPayload') {
        return t.field('count', { type: 'Int', })
      }

      expose({ 
        src: match(type, {
          [PrismaType.Model]: _ => t.model,
          [PrismaType.Query]: _ => t.crud,
          [PrismaType.Mutation]: _ => t.crud,
        }), 
        type,
        fields, 
        blacklist, 
      });
    },
  });
}

export function createSchemaTypes(types, blacklistByName = {}) {
  const uniqueNames = {};
  return types
    .filter(({ name }) => {
      if (uniqueNames[name]) return false;
      
      uniqueNames[name] = 1;
      return true;
    })
    .map(({ name, fields }) => {
      let internalType = PrismaType.Model;
      if (name === 'Query') internalType = PrismaType.Query;
      else if (name === 'Mutation') internalType = PrismaType.Mutation;

      return objType({ 
        name, 
        type: internalType, 
        fields, 
        blacklist: blacklistByName[name] || [],
      })
    });
} 
```

Photon's generated `index.d.ts` still lacks `MutationUpdateManyEmployeeFilter` & `MutationDeleteManyEmployeeFilter` as of prisma2@2.0.0-preview-6, binary version: 4d17a0639165e5a8e54c7e5cf538ffdf114edb

**Comment by Akkuma** [comment]
Same problem exists as of preview 7

**Comment by Akkuma** [comment]
Here is with the latest prisma2:

```
Error: Input Object type MutationUpdateManyEmployeeFilter must define one or more fields.

Input Object type MutationDeleteManyEmployeeFilter must define one or more fields.
    at assertValidSchema (/Users/omni/Documents/git/tabemasu/node_modules/.registry.npmjs.org/graphql/14.4.2/node_modules/graphql/type/validate.js:71:11)
    at assertValidExecutionArguments (/Users/omni/Documents/git/tabemasu/node_modules/.registry.npmjs.org/graphql/14.4.2/node_modules/graphql/execution/execute.js:163:35)
    at executeImpl (/Users/omni/Documents/git/tabemasu/node_modules/.registry.npmjs.org/graphql/14.4.2/node_modules/graphql/execution/execute.js:84:3)
    at Object.execute (/Users/omni/Documents/git/tabemasu/node_modules/.registry.npmjs.org/graphql/14.4.2/node_modules/graphql/execution/execute.js:62:63)
    at Object.generateSchemaHash (/Users/omni/Documents/git/tabemasu/node_modules/.registry.npmjs.org/apollo-server-core/2.8.0_graphql@14.4.2/node_modules/apollo-server-core/dist/utils/schemaHash.js:14:32)
    at ApolloServer.generateSchemaDerivedData (/Users/omni/Documents/git/tabemasu/node_modules/.registry.npmjs.org/apollo-server-core/2.8.0_graphql@14.4.2/node_modules/apollo-server-core/dist/ApolloServer.js:281:41)
    at new ApolloServerBase (/Users/omni/Documents/git/tabemasu/node_modules/.registry.npmjs.org/apollo-server-core/2.8.0_graphql@14.4.2/node_modules/apollo-server-core/dist/ApolloServer.js:203:38)
    at new ApolloServer (/Users/omni/Documents/git/tabemasu/node_modules/.registry.npmjs.org/apollo-server-express/2.8.0_graphql@14.4.2/node_modules/apollo-server-express/dist/ApolloServer.js:46:9)
```

Photon's generated index.d.ts never mentions anything about that https://gist.github.com/Akkuma/2269b691a957ad3e3014e499d7cf7514

**Comment by Akkuma** [comment]
I've been looking through the code and may have a direction to point you in. It looks like Filters & Enums are having some sort of broken interaction at the moment resulting in no fields being generated via `createInputEnumType` and `buildInputObjectType`. The Role enum I created in my schema results in a `RoleFilter` type that has what seems like a broken type definition `export declare type RoleFilter = {};`. The error I'm seeing is on the EmployeeFilter and Employee is the only one that makes use of the Role enum.

**Comment by medv** [comment]
Seeing this issue as well, server boots fine without the Filter-offending prisma-nexus generated mutation. Interestingly, stripping my schema down to bare essentials and re-generating photon/nexus-prisma, and incrementally re-adding deleted schema brings it all back up fine. The issue is that this also creates migrations for the removal/adding steps, which must be saved if any further work is to take place (and dropping the whole database in one of the migrations is not the best thing to have in your version control).

**Comment by jasonkuhrt** [comment]
Hey @Akkuma I think this was fixed by #385. If you still have this issue with the latest versions let me know. Also take a look at the [example](https://github.com/prisma-labs/nexus-prisma/tree/master/examples/blog) as quite a bit has changed (imports etc.) since your initial repro example.

**Comment by gustawdaniel** [comment]
This pull request https://github.com/graphql-nexus/nexus-schema-plugin-prisma/pull/385 is closed but this issue is not solved.

Steps to reproduce:

1) generate prisma client from `shema.prisma`

```
model Account {
  id Int @default(autoincrement()) @id
  name String
  transactions Transaction[]
}

model Transaction {
  id Int @default(autoincrement()) @id
  accountId Int
  account Account @relation(fields: [accountId], references: [id])
}
```

2) run graphql server
